BestFloww/BestFlow
# Star use case

## Pull Request Summary
The backend is complete but I will probably have to change the input format of the put requests when we have the frontend ready.

### Change types


### Testing
- [x ] I have added testing to this PR 


### How to manually test this PR (QA instructions)

You can just CR for now until the frontend is ready. You can also use postman or insomnia to make a put request using api/star route. 
First you have to upload a new transcript. You can't use previous ones because they don't have star field in their schema. Then you can make your put request.
The body of the request should be a JSON looking like this(for now, this might change):
 {
	"question": "some question",
	"projectId": "the id",
	"status" : true
}
You can see the star field is changed from default false to true in the database


## Questions
